web/ViewModels/LandlordProfileModel.cs

```diff
@@ -1,11 +1,14 @@
 ﻿using System.ComponentModel;
 using System.ComponentModel.DataAnnotations;
+using BricksAndHearts.Auth;
 using BricksAndHearts.Database;
 
 namespace BricksAndHearts.ViewModels;
 
 public class LandlordProfileModel
 {
+    public int Id { get; set; }

```

Comment:
Might call this `LandlordId` to differentiate it more clearly from `CurrentUserId` below